hack/testing/test-010-deploy-via-olm-minimal.sh

```diff
@@ -0,0 +1,98 @@
+#!/bin/bash
+# Given an OLM manifest, verify a green field deployment
+# of cluster logging by asserting CLO creates the resources
+# that begets the operands that make up logging.
+
+set -e
+if [ "${DEBUG:-}" = "true" ]; then
+	set -x
+fi
+
+source "$(dirname "${BASH_SOURCE[0]}" )/../lib/init.sh"
+
+os::test::junit::declare_suite_start "${BASH_SOURCE[0]}"
+
+ARTIFACT_DIR=${ARTIFACT_DIR:-"$(pwd)/_output"}
+if [ ! -d $ARTIFACT_DIR ] ; then
+  mkdir -p $ARTIFACT_DIR
+fi
+export NAMESPACE="openshift-logging"
+VERSION=${VERSION:-4.2}
+
+cleanup(){
+  local return_code="$?"
+  set +e
+  oc logs -n ${NAMESPACE} deployment/cluster-logging-operator > $ARTIFACT_DIR/cluster-logging-operator.log 2>&1 ||:
+  oc delete ns ${NAMESPACE} --wait=false --ignore-not-found
+  oc delete crd elasticsearches.logging.openshift.io --wait=false --ignore-not-found
+
+  os::cleanup::all "${return_code}"
+  
+  exit ${return_code}
+}
+trap cleanup exit
+
+if [ -n "${IMAGE_CLUSTER_LOGGING_OPERATOR:-}" ] ; then
+  source "$(dirname $0)/common"
+fi
+if [ -n "${IMAGE_FORMAT:-}" ] ; then
+  IMAGE_CLUSTER_LOGGING_OPERATOR=$(sed -e "s,\${component},cluster-logging-operator," <(echo $IMAGE_FORMAT))
+else
+  IMAGE_CLUSTER_LOGGING_OPERATOR=${IMAGE_CLUSTER_LOGGING_OPERATOR:-quay.io/openshift/origin-cluster-logging-operator:latest}
+fi
+
+KUBECONFIG=${KUBECONFIG:-$HOME/.kube/config}
+
+oc create -n ${NAMESPACE} -f \
+https://raw.githubusercontent.com/coreos/prometheus-operator/master/example/prometheus-operator-crd/prometheusrule.crd.yaml || :
+oc create -n ${NAMESPACE} -f \
+https://raw.githubusercontent.com/coreos/prometheus-operator/master/example/prometheus-operator-crd/servicemonitor.crd.yaml || :
+
+repo_dir="$(dirname $0)/../.."
+oc create ns ${NAMESPACE} || :
+
+oc create -f ${repo_dir}/vendor/github.com/openshift/elasticsearch-operator/manifests/${VERSION}/elasticsearches.crd.yaml ||:
+
+manifest=${repo_dir}/manifests
+os::log::info "Deploying operator from ${manifest}"
+NAMESPACE=${NAMESPACE} \
+VERSION=${VERSION} \
+OPERATOR_IMAGE=${IMAGE_CLUSTER_LOGGING_OPERATOR} \
+MANIFEST_DIR=${manifest} \
+TEST_NAMESPACE=${NAMESPACE} \
+TARGET_NAMESPACE=${NAMESPACE} \
+${repo_dir}/hack/vendor/olm-test-script/e2e-olm.sh
+
+if [ "$?" != "0" ] ; then
+	os::log::error "Error deploying operator via OLM using manifest: $manifest"
+	exit 1
+fi
+
+TIMEOUT_MIN=$((2 * $minute))
+
+# wait for operator to be ready
+os::cmd::try_until_text "oc -n $NAMESPACE get deployment cluster-logging-operator -o jsonpath={.status.availableReplicas} --ignore-not-found" "1" ${TIMEOUT_MIN}
```

Comment:
I get the cluster-logging-operator - but I don't get an elasticsearch-operator - should I?

Reply:
hmm - I guess this test doesn't need EO - for a real test, will need to use olm-test-script to deploy EO

Reply:
No.  This test is simply to verify the CLO deployment and that it creates the resources we expect.  It intentionally is simple like the legacy test.  I would expect we add a broader set of tests like OAL test suite or something that actually verifies pods spin up, etc.